index.html

```diff
@@ -10,20 +10,25 @@
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Inter&family=Playfair+Display&display=swap" rel="stylesheet">
+  <script src="script.js"></script>
+
   <title>Aly Atalla</title>
 </head>
 
 <body>
   <header>
     <nav class="nav">
-      <div class="menu-button">
-        <i class="fa-solid fa-bars fa-xl"></i>
-      </div>
+      
       <ul class="header-links">
-        <li class="header-link" href="#works">Experience</li>
-        <li class="header-link" href="#about">About</li>
-        <li class="header-link" href="#contact">Contact</li>
+        <li class="header-link" data-target="#works">Experience</li>
+        <li class="header-link" data-target="#about">About</li>
+        <li class="header-link" data-target="#contact">Contact</li>
```

Comment:
- [ ] When clicking the navigation links the user is not redirected to the correct part of the page, in order to make this work you should add an anchor tag with an href directing to the id of the section you want to display.